unlock-js/src/__tests__/helpers/walletServiceHelper.js

```diff
@@ -133,7 +133,6 @@ export const prepContract = ({
       networkId: 0,
       nonce: 0,
       value: utils.bigNumberify(encodedValue),
-      gasPrice: utils.bigNumberify(testParams.gas),
```

Comment:
This is actually not required and probably bad practice: the wallet should set the gas price.